.github_changelog_generator

```diff
@@ -1 +1 @@
-add-sections= {"major":{"prefix":"### Major Changes","labels":["major"]}, "renovate":{"prefix":"### Minor Changes","labels":["renovate", "patch"]}, "breaking":{"prefix":"### Breaking Changes","labels":["breaking"]}}
+add-sections= {"major":{"prefix":"### Minor Changes","labels":["major"]}, "renovate":{"prefix":"### Patch Changes","labels":["renovate", "patch"]}, "breaking":{"prefix":"### Breaking Changes","labels":["breaking"]}}
```

Comment:
Now you are labelling major changes as minor changes, don't you?

Reply:
true. I made a new suggestion